docs/cloud/integrations/datagrip.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Connect to Steampipe Cloud from DataGrip
+sidebar_label: DataGrip
+---
+## Connect to Steampipe Cloud from DataGrip
+
+[DataGrip](https://www.jetbrains.com/datagrip/) is a cross platform SQL IDE to query, edit and manage your databases.
+
+Steampipe provides a single interface to all your cloud, code, logs and more. Because it's built on Postgres, Steampipe provides an endpoint that any Postgres-compatible client -- including DataGrip -- can connect to.
+
+The [Connect](/docs/cloud/integrations/overview) tab for your workspace provides the details you need to connect DataGrip to Steampipe Cloud.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/steampipe-cloud-connect-details.jpg" />
+</div>
+
+Once the connection to Steampipe cloud is established and tested, you can explore the tables provided by the Steampipe plugins, run queries and build reports.
+
+You can also connect DataGrip to [Steampipe CLI](https://steampipe.io/downloads). To do that, run `steampipe service start --show-password` and use the displayed connection details.
+
+```
+Steampipe service is running:
+
+Database:
+
+  Host(s):            localhost, 127.0.0.1, 192.168.29.204
+  Port:               9193
+  Database:           steampipe
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+  ```
+
+## Getting started
+
+[DataGrip](https://www.jetbrains.com/datagrip/download/#section=mac) is available to use on the desktop and has a 30 day trial version after which a license is required. In this example we will create a Steampipe Cloud connection with DataGrip and query the AWS availability zones.
+
+To create a new connection click on `New` and add a PostgreSQL data source. Click `Download missing driver files` at the bottom of the settings page, since DataGrip does not include bundled drivers required to interact with the database. Enter the Steampipe Cloud connection details and click `Test Connection` to Verify.
```

Comment:
```suggestion
To create a new connection click on `New` and add a PostgreSQL data source. Click `Download missing driver files` at the bottom of the settings page, since DataGrip does not include bundled drivers required to interact with the database. Enter the Steampipe Cloud connection details and click `Test Connection` to `Verify`.
```

Reply:
Rectified